wradlib/georef/xarray.py

```diff
@@ -109,18 +109,18 @@ def create_xarray_dataarray(
 
     if phi is None:
         if sweep_mode == "azimuth_surveillance":
-            phi = np.arange(data.shape[0], dtype=np.float_)
+            phi = np.arange(data.shape[0], dtype=np.float64)
```

Comment:
I'm not sure but would `float` be enough? Or is it better to explicitely ask for `float64`. I'm not that deep into this.